mne/beamformer/tests/test_rap_music.py

```diff
@@ -0,0 +1,144 @@
+# Authors: Yousra Bekhti <yousra.bekhti@gmail.com>
+#          Alexandre Gramfort <alexandre.gramfort@telecom-paristech.fr>
+#
+# License: BSD (3-clause)
+
+import os.path as op
+import numpy as np
+from scipy import linalg
+
+import warnings
+from nose.tools import assert_true
+
+import mne
+from mne.datasets import testing
+from mne.beamformer import rap_music
+
+
+data_path = testing.data_path(download=False)
+fname_ave = op.join(data_path, 'MEG', 'sample', 'sample_audvis-ave.fif')
+fname_cov = op.join(data_path, 'MEG', 'sample', 'sample_audvis_trunc-cov.fif')
+fname_fwd = op.join(data_path, 'MEG', 'sample',
+                    'sample_audvis_trunc-meg-eeg-oct-4-fwd.fif')
+
+warnings.simplefilter('always')  # enable b/c these tests throw warnings
+
+
+def read_forward_solution_meg(fname_fwd, **kwargs):
```

Comment:
this one isn't actually run, it's just used by the other tests (that have one-line descriptions)